### PR TITLE
SAAS-1000 Update documentation about the ACR values

### PR DESCRIPTION
**Pull request by czarnecki-piotr**

### CODE REVIEW

File: docs/products/access/topics/oidc/overview/scopes-and-claims.md [hunk 1]
```diff
@@ -44,32 +44,47 @@ identify their source and also prevent from potential clashes/collisions.
...
-The table below summarizes currently supported values. The available values are also exposed via the [Discovery API](../../../api-reference/oidc/discovery.md).
+For CIM IdP, only a single ACR value can be specified. Sending multiple values will result in a `Bad Request` error.
 
-| ACR value                                   | Description                                                                      |
-|---------------------------------------------|----------------------------------------------------------------------------------|
-| urn:onegini.com:oidc:authentication_level:1 | Requires an authentication level of at least 1 from Onegini CIM.  |
-| urn:onegini.com:oidc:authentication_level:2 | Requires an authentication level of at least 2 from Onegini CIM.  |
-| urn:onegini.com:oidc:authentication_level:3 | Requires an authentication level of at least 3 from Onegini CIM.  |
-| urn:onegini.com:oidc:authentication_level:4 | Requires an authentication level of at least 4 from Onegini CIM.  |
+#### OAuth
```
Comment (rvroon) [inline docs/products/access/topics/oidc/overview/scopes-and-claims.md:67]: please add OIDC here as well

File: docs/products/access/topics/oidc/overview/scopes-and-claims.md [hunk 2]
```diff
@@ -44,32 +44,47 @@ identify their source and also prevent from potential clashes/collisions.
...
+
+For OAuth-based identity providers (e.g. Tulip, Identity Broker), requested ACR values are forwarded upstream to the configured identity
+provider.
+
+Supported ACR values from IdP OpenID configuration are exposed via the [Discovery API](../../../api-reference/oidc/discovery.md). However,
+Access does not validate provided ACR values and allows those not in OpenID configuration.
```
Comment (rvroon) [inline docs/products/access/topics/oidc/overview/scopes-and-claims.md:73]: This is not completely clear to me. Be more clear on what we expect from the external IDP and what Access does. Also we may need to be explicit that not all IdP types support all ACR values listed in the discovery endpoints.

File: docs/products/access/topics/oidc/overview/scopes-and-claims.md [hunk 3]
```diff
@@ -44,32 +44,47 @@ identify their source and also prevent from potential clashes/collisions.
...
 To request a specific ACR value include `acr_values` parameter when requesting ID Token, e.g.:
+
 ```http
 http://tokenserver.example.com/oauth/authorize?response_type=code&client_id=openid&redirect_uri=http%3A%2F%2F.example.com&scope=openid&state=d5dbda85-ecdb-4172-9ada-7ba15c6982d0&acr_values=urn:onegini.com:oidc:authentication_level:2
 ```
 
-Currently, only a single ACR value can be specified at a time. Sending multiple values will result in a `Bad Request` error.
-Onegini Access may return an ACR with a value that is higher than the authentication level that was requested.
-
+Access may return an ACR with a value that is higher than the authentication level that was requested.
```
Comment (rvroon) [inline docs/products/access/topics/oidc/overview/scopes-and-claims.md:81]: Are you sure? I think the spec says that you have to provide one of the ACR values that is requested 🤔

**Review by rvroon — CHANGES_REQUESTED** [review]

**Review by osvso — DISMISSED** [review]

File: docs/products/access/topics/oidc/overview/scopes-and-claims.md (hunk 2 above)
Comment (czarnecki-piotr) [reply docs/products/access/topics/oidc/overview/scopes-and-claims.md]: Please check if it's clear now

File: docs/products/access/topics/oidc/overview/scopes-and-claims.md (hunk 3 above)
Comment (czarnecki-piotr) [reply docs/products/access/topics/oidc/overview/scopes-and-claims.md]: It's certainly true for CIM, so I moved this sentence to the CIM section.

**Review by osvso — APPROVED** [review]

**Pull request merged**